tests/components/ModalTest.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, fireEvent, waitFor, screen } from '@testing-library/react';
+import { Modal } from '../../packages/modal/src/component';
+
+describe('Modal', () => {
```

Comment:
Should we test component classes strings in all Warp components, too, or does it not make sense?

Reply:
I thought handling tests before the cleanup tasks was supposed to help the reviewers see which classes were changed as a result of the cleanup 😊 I also saw you added similar tests in other components, so I was wondering if that should be applied to all components or only to some. But sure, this can be added as part of the classes cleanup task 👍

Reply:
Modal discussion with designers is ongoing, so if the paddings are changed, we should revisit this before doing a component cleanup :)